tests/core/tests/test_resources.py

```diff
@@ -1,3 +1,5 @@
+# coding: utf-8
```

Comment:
No need for the encoding here.

Reply:
My editor will add this in the beginning of each file that contains non-ascii charecters. I think it is because of this line:
`raise ValueError("Ова вриједност је страшна!")`
If you don't like it, I can remove it:)